### PR TITLE
Updates for dcw 2.0.1

### PR DESCRIPTION
**Pull request by Esteban82**
I made all the modifications of the latest issues. They are describe in ChangeLog

Checksum: 
44594a0b190fc1bfd70fad80215a9013  dcw-gmt-2.0.1.tar.gz

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -2,6 +2,17 @@
         Contains changes made to the data files under dcw-gmt/orig etc
 -----------------------------------------------------------------------------------------
 
+2020-09-12      esteban82
+	        New 2.1.0. Fix some names (North Macedonia, Eswatini, Cabo Verde,
```
Comment (seisman) [inline ChangeLog:6]: 2.0.1 or 2.1.0?
```suggestion
2021-09-12      esteban82
	        New 2.0.1. Fix some names (North Macedonia, Eswatini, Cabo Verde,
```

**Comment by PaulWessel** [comment]
I would like to check these out by switching to the branch but I think you made a branch in your own account and not in GenericMappingTools? So how can I get the files so I can check and build?

**Comment by Esteban82** [comment]
Yes Paul. I noticed that. Could be it possible that I need some special authorization? Because I make the PR as always and a message appear saying that a personal fork would be created.

File: ChangeLog (hunk 1 above)
Comment (Esteban82) [reply ChangeLog:6]: I think it will be 2.0.1.

**Comment by PaulWessel** [comment]
OK, I have added the GMT core developers to have write access to the dcw-gmt repo.  I am not sure what you need to do now, i.e., checkout another copy and merge locally? Maybe @seisman knows if you do not know.

**Comment by Esteban82** [comment]
If you want I can send the tar file trough Dropbox?

**Comment by seisman** [comment]
image

Click the "Open with" button, and you will see the instructions to check out this PR using git commands.

**Comment by PaulWessel** [comment]
OK, you could send me the tarball via dropbox and I can install and run tests.

**Comment by Esteban82** [comment]
https://www.dropbox.com/s/l3adhhiwm8qcydm/dcw-gmt-2.0.1.zip?dl=0

**Review by PaulWessel — APPROVED** [review]

**Comment by PaulWessel** [comment]
Looks great!

**Comment by Esteban82** [comment]
Closes #14,  #17, #19.

**Comment by PaulWessel** [comment]
Hi @Esteban82.  Sorry, I dropped the ball on this.  I guess there is no reason to not release this on asap?

**Comment by Esteban82** [comment]
Don't worry. I don't how this really works. I have no rush. I can wait for 6.3 I think.

**Pull request merged**